compose/docker-compose.yml

```diff
@@ -28,7 +27,8 @@ services:
       MQ_PORT: "443"
       MQ_SERVER_PORT: "1883"
       VERBOSITY: "1"
-      MQ_ADMIN_PASSWORD: "REPLACE_MQ_ADMIN_PASSWORD"
+      MQ_PASSWORD: "REPLACE_MQ_PASSWORD"
```

Comment:
these changes are also applicable to docker-compose.ee.yml and docker-compose.reference.yml

Reply:
that explains why my setup isn't working. I am testing on ee.

Reply:
docker-compose.ee.yml actually needs a few things updated. there's no stun setup on there either.

Reply:
updated docker-compose.ee.yml and I believe docker-compose.reference.yml is being updated by @afeiszli